pkg/cluster/describe.go

```diff
@@ -113,6 +113,18 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		computesStr = fmt.Sprintf("%d", cluster.Nodes().Compute())
 	}
 
+	clusterAdminEnabled := false
```

Comment:
This entire code should go inside `func isClusterAdminEnabled`.
We will also need it for user add/delete commands.

Reply:
Well. Not that critical. I see you will need to pass the subscription response.
We can refactor later when it's actually used.

Reply:
We don't have any validation if cluster admin is enabled in add/delete commands and we are relying on the backend validation. Having said that, we can refactor in a separate change.